src/OutputProviders/FFProbeProvider.php

```diff
@@ -34,7 +34,7 @@ public function getOutput()
         }
 
         // File doesn't exist
-        if (!file_exists($this->movieFile)) {
+        if (@file_get_contents($this->movieFile, false, null, 0, 1) === false) {
```

Comment:
`0/0` as `offset/length` is supported as well. Let's use this one as it allows to process empty files as well.
```suggestion
        if (@file_get_contents($this->movieFile, false, null, 0, 0) === false) {
```